practical/bash/import_as_create_annotation.sh

```diff
@@ -25,14 +25,19 @@
 #
 #
 dir=$(pwd -P)
-OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
-PASSWORD=${PASSWORD:-ome}
-HOST=${HOST:-outreach.openmicroscopy.org}
+#OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+#PASSWORD=${PASSWORD:-ome}
+#HOST=${HOST:-outreach.openmicroscopy.org}
 SUDOER=${SUDOER:-trainer-1}
-OMEUSER=${OMEUSER:-user-1}
+#OMEUSER=${OMEUSER:-user-1}
 IMAGEPATH=${IMAGEPATH:-}
 ATTACHMENTPATH=${ATTACHMENTPATH:-}
 
+OMEROPATH=${OMEROPATH:-/Users/jmarie/Documents/git-repo/openmicroscopy/dist/bin/omero}
```

Comment:
``/Users/jmarie/...`` cannot be right -> consequently, it does not work on outreach either